### PR TITLE
[DNS] Migrated + streamlined record-specific info

### PR DESCRIPTION
**Pull request by kodster28**
Partially addresses PCX-3219

### CODE REVIEW

File: products/dns/src/content/manage-dns-records/reference/dns-record-types.md [hunk 1]
```diff
@@ -5,4 +5,112 @@ pcx-content-type: reference
```
Comment (kodster28) [inline products/dns/src/content/manage-dns-records/reference/dns-record-types.md:5]: Not all of these records are detailed in a parallel manner, but I think that's the best we can do for now.

There are different levels of details needed for each record + different levels of info available in the learning center.

File: products/dns/src/content/manage-dns-records/reference/dns-record-types.md [hunk 2]
```diff
@@ -5,4 +5,114 @@ pcx-content-type: reference
...
+
+You can use CNAME records to point to other CNAME records (`www.example2.com` --> `www.example1.com` --> `www.example.com`), but the final record must point to a hostname with a valid IP address (and therefore a valid A or AAAA record).
+
+---
+
+## Email authentication and domain validation
+
+These records are recommended regardless of whether your domain sends email messages. Creating [negative email records](https://blog.cloudflare.com/tackling-email-spoofing/) can help protect your domain against email spoofing.
+
+### MX
```
Comment (dcpena) [inline products/dns/src/content/manage-dns-records/reference/dns-record-types.md:56]: For this and the following acronyms, are they spelled out anywhere else? If not, I'd consider spelling it out in the intro sentence after the header with the acronym in parentheses.

File: products/dns/src/content/manage-dns-records/reference/dns-record-types.md [hunk 3]
```diff
@@ -5,4 +5,114 @@ pcx-content-type: reference
...
+
+MX records are necessary for delivery of email to a mail server:
```
Comment (dcpena) [inline products/dns/src/content/manage-dns-records/reference/dns-record-types.md]: You use a colon here but for SPF and DMARC, you use a period. I'd choose one and update the others for consistency.

**Review by dcpena — APPROVED** [review]
Two non-blocking comments

**Pull request merged**